ingestion/Makefile

```diff
@@ -11,23 +11,23 @@ endif
 
 .PHONY: install
 install:  ## Install the ingestion module to the current environment
-	python -m pip install $(INGESTION_DIR)/
+	PIP_CONSTRAINT=$(INGESTION_DIR)/constraints.txt python -m pip install $(INGESTION_DIR)/
 
 .PHONY: install_dev
 install_dev:  ## Install the ingestion module with dev dependencies
-	python -m pip install "$(INGESTION_DIR)[dev]/"
+	PIP_CONSTRAINT=$(INGESTION_DIR)/constraints.txt python -m pip install "$(INGESTION_DIR)[dev]/"
 
 .PHONY: install_test
 install_test:  ## Install the ingestion module with test dependencies
-	python -m pip install "$(INGESTION_DIR)[test]/"
+	PIP_CONSTRAINT=$(INGESTION_DIR)/constraints.txt python -m pip install "$(INGESTION_DIR)[test]/"
 
 .PHONY: install_all
 install_all:  ## Install the ingestion module with all dependencies
-	python -m pip install "$(INGESTION_DIR)[all]/"
+	PIP_CONSTRAINT=$(INGESTION_DIR)/constraints.txt python -m pip install "$(INGESTION_DIR)[all]/"
 
 .PHONY: install_apis
 install_apis:  ## Install the REST APIs module to the current environment
-	python -m pip install $(ROOT_DIR)/openmetadata-airflow-apis/ setuptools==69.0.2
+	PIP_CONSTRAINT=$(ROOT_DIR)/openmetadata-airflow-apis/constraints.txt python -m pip install $(ROOT_DIR)/openmetadata-airflow-apis/ setuptools==69.0.2
```

Comment:
why are we adding these constraints here? Can't we handle it from the setup file?

Reply:
or in pyproject.toml I think we have a requires https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/pyproject.toml#L2

I'd like to avoid having to look at versions to yet another place

Reply:
tried different solutions here, pyproject.toml setuptools version modification didn't work as expected, we already have that in place and it is still failing, the reach for requires doesn't impact 3rd party packages as far as i could test for the setuptools issue

Reply:
only solution that could impact 3rd party packages was the constraint one

Reply:
There is one more solution that i have yet to try i.e. --no-build-isolation

Reply:
ok thanks, let's keep it as-is until setuptools fixes it on their end